python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -477,7 +596,7 @@ def __new__(
     ):
         check.opt_list_param(run_requests, "run_requests", RunRequest)
         check.opt_str_param(skip_message, "skip_message")
-        check.opt_str_param(cursor, "cursor")
+        # check.opt_str_param(cursor, "cursor")
```

Comment:
add this check back once i figure out how to handle the `mapping[assetkey, str]` type for the new context